ApiFirst/AppServices/Services/OrderAppService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using AppModels.AppModels.Orders;
+using AppServices.Interfaces;
+using AutoMapper;
+using DomainModels.Models;
+using DomainServices.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace AppServices.Services
+{
+    public class OrderAppService : IOrderAppService
+    {
+        private readonly IOrderService _orderServices;
+        private readonly IMapper _mapper;
+
+        public OrderAppService(IMapper mapper, IOrderService orderServices)
+        {
+            _mapper = mapper ??
+                throw new ArgumentNullException(nameof(mapper));
+            _orderServices = orderServices ??
+                throw new ArgumentNullException(nameof(orderServices));
+        }
+
+        public async Task<long> CreateAsync(CreateOrder model)
+        {
+            var order = _mapper.Map<Order>(model);
+            return await _orderServices.CreateAsync(order);
```

Comment:
Adicionar nos outros métodos da classe
```suggestion
            return await _orderServices.CreateAsync(order).ConfigureAwait(false);
```